api/internal/conf/conf.go

```diff
@@ -249,3 +254,16 @@ func initEtcdConfig(conf Etcd) {
 		Prefix:    prefix,
 	}
 }
+
+// initialize parallelism settings
+func initParallelism(choiceCores int) {
+	if choiceCores < 0 {
```

Comment:
how about
```suggestion
	if choiceCores < 1 {
```

because 0 means no limit.

Reply:
True. It makes sense.